<html>
  <head>
    <meta name=viewport content="width=800">
 <meta name="generator" content="HTML Tidy for Linux/x86 (vers 11 February 2007), see www.w3.org">

  <style type="text/css">
    /* Color scheme stolen from Sergey Karayev */
  .nav li {
  font-family: 'Lato', Verdana, Helvetica, sans-serif;
  font-size: 18px;
  font-color: white;
  margin: 0px;
  padding: 0px;
  display: inline;
  }
  .nav ul{
    list-style: none;
    background-color: white;
    text-align: center;
    padding: 0;
    margin: 0;
  }

  .nav a {
  font-color: white;
  display: block;
  padding: 10px 20px;
  }


  @media screen and (min-width: 600px) {
    .nav li {
      width: 120px;
      border-bottom: none;
      height: 50px;
      line-height: 50px;
      font-size: 1.4em;
    }

    /* Option 1 - Display Inline */
    .nav li {
      display: inline-block;
      margin-right: -4px;
    }



    a {
      color: #1772d0;
      text-decoration: none;
    }

    a:focus,
    a:hover {
      color: #f09228;
      text-decoration: none;
    }

    body,
    td,
    th,
    tr,
    p,
    a {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px
    }

    strong {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px;
    }

    heading {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 22px;
    }

    papertitle {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px;
      font-weight: 700
    }

    name {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 32px;
    }

    .one {
      width: 160px;
      height: 160px;
      position: relative;
    }

    .two {
      width: 160px;
      height: 160px;
      position: absolute;
      transition: opacity .2s ease-in-out;
      -moz-transition: opacity .2s ease-in-out;
      -webkit-transition: opacity .2s ease-in-out;
    }

    .fade {
      transition: opacity .2s ease-in-out;
      -moz-transition: opacity .2s ease-in-out;
      -webkit-transition: opacity .2s ease-in-out;
    }

    span.highlight {
      background-color: #ffffd0;
    }
  </style>
  <title>张小蕾 Xiaolei Zhang </title>
   <meta http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <link href='https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
</head>


  <div class="nav">
    <ul>
      <li class="active"><a href="#about">About</a></li>
      <li><a href="#publication">Publication</a></li>
      <li><a href="#code">Code Repository</a></li>
     <!-- <li><a href="#misc">Misc</a></li> -->
    </ul>
  </div>


  <body>

<div class="row" id="about">
<table width="800" border="0" align="center" cellspacing="0" cellpadding="0">
    <tr>
      <td>
<table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
     <tr>
       <td width="67%" valign="middle">
         <p align="center">
           <name>Xiaolei Zhang 张小蕾 </name>
           
           
            <p>I am passionate about advancing healthcare using computational and statistical approaches. I am currently a Postdoctoral Fellow based at European Bioinformatics Institute (Hinxton, UK). I am also awared as a <a href=https://www.postdocacademy.cam.ac.uk/2021-2022-borysiewicz-interdisciplinary-fellows>Borysiewicz Interdisplinary Fellow</a> (University of Cambridge) and Junior Research Fellow (Darwin College, University of Cambridge). Previously I  studied Computational Biology and Bioinformatics in University of Science and Technology of China, ETH Zurich and finished my PhD in Imperial College London in the study of statistical methods for clinical genome interpretation with specific application in inherited cardiac conditions.
              </p>
         </p>
             </td>
        <p>     
              </p>
        
         <td width="33%">
              <img src="images/portait_xz.png">
            </td>
            </tr>
  
          </table>
</div>

<div class="row" id="Publications">


<table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="100%" valign="middle">
              <heading>Selected Publications</heading>
            </td>
          </tr>
        </table>
<table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="67%" valign="middle">
              All publications are listed in <a href="https://scholar.google.com/citations?user=dtB5VmgAAAAJ&hl=en">Google Scholar</a> 

<p><b>X Zhang</b> <i>et al.</i> Population-scale analysis of inheritance patterns across 858,635 individuals reveals recent historical migration patterns across the North Sea from the Middle Ages to the Industrial Revolution. <i>bioRxiv</i> <a target href = "https://www.biorxiv.org/content/10.1101/2025.03.19.643007v1.full">[Manuscript]</a><a target href = "https://github.com/XiaoleiZ/UK_DNK_IBD">[Code]</a></p>
<p>Laura Harris, Ellen M McDonagh, <b>X Zhang<sup>*<sup></b>,<i>et al.</i>Tomas Fitzgerald. Genome-wide association testing beyond SNPs. <i>Nature Reviews Genetics</i> <a target href = "https://www.nature.com/articles/s41576-024-00778-y">[Publication]</a>(* co-first author)</p> 
<p><b>X Zhang</b>, PI Theotokis, N Li, C Wright, KE Samocha, N Whiffin, JS Ware. Genetic constraint at single amino acid resolution in protein domains improves missense variant prioritisation and gene discovery Genome Medicine <a target href="https://doi.org/10.1186/s13073-024-01358-9">[Publication]</a><a target href= "https://www.cardiodb.org/hmc/">[Website]</a><a target href = "https://github.com/ImperialCardioGenetics/homologous-missense-constraint">[Code]</p>       
<p><b>X Zhang</b>, M Wakeling, J Ware, N Whiffin Annotating high-impact 5’untranslated region variants with the UTRannotator <i>Bioinformatics</i> <a target href = "https://doi.org/10.1101/2020.06.03.132266">[Publication]</a> <a target href ="https://github.com/ImperialCardioGenetics/UTRannotator"> [Code]</a></p>                
<p><b>X Zhang</b> <i>et al.</i> Disease-specific variant pathogenicity prediction significantly improves clinical variant interpretation in inherited cardiac conditions. <i>Genetics in Medicine</i> <a target href = "https://www.nature.com/articles/s41436-020-00972-3">[Publication]</a> <a target href = "https://www.cardiodb.org/cardioboost"> [Website]</a><a target href = "https://github.com/ImperialCardioGenetics/CardioBoost_manuscript"> [Website]</a></p>
<p><b>X Zhang</b>, E Minikel, A O’Donnell-Luria, D MacArthur, J Ware, B Weisburd. ClinVar Data Parsing <i>. Wellcome Open Research</i> <a target href = "https://wellcomeopenresearch.org/articles/2-33/v1">[Publication]</a><a target href = "https://doi.org/10.5281/zenodo.399052">[Code]</a></p>
<p>Y Dong, <b>X Zhang<sup>*<sup></b>, M Xie, B Arefnezhad  <i>et al.</i> Reference genome of wild goat (capra aegagrus) and sequencing of goat breeds provide insight into genic basis of goat domestication. <i>BMC genomics</i> <a target href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1606-1">[Publication]</a> (* co-first author)</p>
<p>K McGurk, <b>X Zhang</b>, et.al (2023). The penetrance of rare variants in cardiomyopathy-associated genes: A cross-sectional approach to estimating penetrance for secondary findings. <i>American Journal of Human Genetics</i> <a target href = "https://doi.org/10.1016/j.ajhg.2023.08.003">[Publication]</a></p>
<p>N Whiffin, KJ Karczewski, <b>X Zhang</b>, S Chothani, <i>et al.</i> Characterising the loss-of-function impact of 5' untranslated region variants in whole genome sequence data from 15,708 individuals. <i>Nature Communications</i> <a target href = "https://www.nature.com/articles/s41467-019-10717-9">[Publication]</a> <a target href = "https://github.com/ImperialCardioGenetics/uORFs/tree/master/5primeUTRannotator"> [Code] </a></p>
<p>M Maiolo, <b>X Zhang</b>, M Gil, M Anisimova. Progressive multiple sequence alignment With indel evolution. <i>BMC bioinformatics</i> <a target href = "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2357-1">[Publication]</a></p>
            </td>
          </tr>
        </table>

        <div class="row" id="Handy scripts/resources">
          <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
                     <tr>
                       <td width="100%" valign="middle">
                         <heading>Code Repository</heading>
                         <p>Here is a collection of code repositories I've worked on</p>
                         <ul>
                           <li><p>
                           <a target href = https://github.com/XiaoleiZ/UK_DNK_IBD>IBDbillion</a>: scalable and efficient C program to analyse billion records of identity-by-descent data
                           </p></li>
                           <li><p>
                             <a target href=https://github.com/ImperialCardioGenetics/CardioBoost_manuscript>Data anslysis code for CardioBoost manuscript</a>
                           </p></li>
                           <li><p>
                             <a target href=https://github.com/ImperialCardioGenetics/homologous-missense-constraint>Data anslysis code for Homologous missense constraint manuscript</a>
                           </p></li>
                 <!--         <li><p>
                          <a target href=https://github.com/XiaoleiZ/Reproduce_HilbertCNN/>Reproduce HilbertCNN</a>: An image representation based convolutional network for DNA classification. 
                           </p></li>-->
                           <li><p>
                          <a target href=https://github.com/macarthur-lab/clinvar>ClinVar Data Parsing</a>: an automatic data retreival pipeline of ClinVar database for variant interpretation study 
                           </p></li>
                           <li><p>
                          <a target href=https://github.com/ImperialCardioGenetics/UTRannotator>5' UTR Annotator</a>: an VEP plugin to annoate 5' UTR variant effect 
                           </p></li>
                           <li><p>
                           <a target href = https://github.com/XiaoleiZ/parse_pfam_stockholm>Parsing the Massive Pfam Stockholm Alignment file
                           </p></li>
                            <!--    <li><p>
                          <a target href= https://xiaoleiz.github.io/one_papar_per_day/>One Paper Per Day Challenge</a> 
                           </p></li>  -->
                           </td>
                             </tr>
                             </table>
        </div>

<!--     <div class="row" id="misc">
      <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
        <heading>Misc</heading>
      <tr>
                    
                    <td width = "30%" valign="middle">
                     <p><i>Books I've read in 2019</i></p>
                     <p><i>Animal Liberation (Peter Singer)</i></p>
                   </td>
                   <td width = "70%">
                     <p>Summary</p>
                     <p>Rethinking the ethic status of animals</p>
                   </td>
                 </tr>
               </table>

        </div> -->

            </td>
          </tr>
        </table>
</div>



  <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
  <tr>
    <td width="80%" valign="middle">
      </td>
    <td width="20%">
      <p><a target href=https://jonbarron.info>(I like this minimalist website)</a></p>
      </td>
        </tr>
      </table>


</body>
  </html>
